### PR TITLE
Increasing some earnings.

### PR DESCRIPTION
**Pull request by Specifer**
I need your word if the value here is presented good. This PR is for issue #577

Thank you for making a contribution to [Las Venturas Playground](https://sa-mp.nl/)!

Please take a moment to fill in the questions in this template, to make it easier for our developers
to understand what you're doing.

## What is being changed?
Value of earnings

## Why is this being changed?
For an issue

## How are you making the change?
  [ ] I have a working check-out of Las Venturas Playground. Done 
  [ ] I have included tests for any changed JavaScript code. Done
  [ ] I have tested this change myself on my local server. Done

### CODE REVIEW

File: pawn/Interface/Economy.pwn [hunk 1]
```diff
@@ -124,10 +124,10 @@ GetEconomyValue(EconomyValueType: type, inputValue = 0) {
         case TeleportWithoutVehicle:            return 1000;
         case TuneCommand:                       return 1250;
         case VehicleCrusherReward:              return 2000;
-        case VehicleExportReward:               return floatround(inputValue /* vehicle health [371-1000] */ * 15 * 1.216);
-        case VehicleExportRewardOwnerShare:     return floatround(inputValue /* vehicle health [371-1000] */ * 15 * 1.216 * 0.1);
-        case WantedLevelAward:                  return 1000 * inputValue /* wanted level stars */;
-        case WantedLevelOwnerShare:             return 10 * inputValue /* wanted level stars */;
+        case VehicleExportReward:               return floatround(inputValue /* vehicle health [371-1000] */ * 60 * 1.45);
```
Comment (RussellLVP) [inline pawn/Interface/Economy.pwn]: We're increasing this by well over five times. Can we reduce that a little bit, and change the `60` to `45`?

Comment (Specifer) [reply pawn/Interface/Economy.pwn]: Ah Well I have different thoughts. We are reducing this max earning from 87000 to 65250. Are you definitely sure we should reduce it a bit to 45?

Comment (Specifer) [reply pawn/Interface/Economy.pwn]: @RussellLVP, Done it !! Reduced it to 45.

**Review by RussellLVP — APPROVED** [review]
Great, thank you Specifer!

**Pull request merged**